tasks/Task.cpp

```diff
@@ -211,8 +204,9 @@ void Task::stopHook()
     TaskBase::stopHook();
 }
 
-// void Task::cleanupHook()
-// {
-//     TaskBase::cleanupHook();
-// }
+void Task::cleanupHook()
+{
+    TaskBase::cleanupHook();
+    mDriver.release();
```

Comment:
Note that release does not delete the object but releases its ownership. I guess you want to call reset here.

Reply:
Seems right!
That should also be changed here:
https://github.com/rock-core/drivers-orogen-iodrivers_base/blob/master/README.md

Reply:
True!